PL2/PL2-gui/src/main/java/nl/tudelft/pl2016gr2/gui/view/graph/DrawComparedGraphs.java

```diff
@@ -256,18 +299,37 @@ private static void drawEdges(Pane pane, ArrayList<GraphNodeOrder> graph, int st
   }
 
   /**
+   * Calculate the edge width.
+   *
+   * @param maxGenomes the total amount of genomes in the graph.
+   * @param from       the node from which the edge comes.
+   * @param to         the node to which the edge goes.
+   * @return the edge width.
+   */
+  public static double calculateEdgeWidth(int maxGenomes, AbstractNode from, AbstractNode to) {
+    int genomesOverEdge = from.getGenomesOverEdge(to);
+    double edgeWith = Math.log(100.0 * genomesOverEdge / maxGenomes) * 0.8;
```

Comment:
Simply put some comments around these magic numbers explaining what they do and why they're set to this value. Putting them in constants wouldn't actually help with understanding them, only move the `WTF?` moment to the constant definition rather than where the constant is used.